WalletWasabi.Gui/Global.cs

```diff
@@ -119,87 +119,101 @@ public Global(string dataDir, Config config, UiConfig uiConfig, WalletManager wa
 
 		private TaskCompletionSource<bool> InitializationCompleted { get; } = new();
 
-		private bool InitializationStarted { get; set; } = false;
+		private volatile bool _disposeRequested;
```

Comment:
StoppingCts.IsCancellationRequested equal to _disposeRequested? 
Remove _disposeRequested?

Reply:
We dispose `StoppingCts` and then we could fail with an "DisposedObject" exception. So I come up with `_disposeRequested`.

Reply:
I have added explaining comments to make it more obvious why it is as it is.

Reply:
Everything should be canceled before StoppingCts disposal, shouldn't it?

Reply:
This is an app lifetime Cts, do we need to dispose it at all? 

https://github.com/aspnet/AspNetKatana/issues/108
https://stackoverflow.com/questions/6960520/when-to-dispose-cancellationtokensource

Reply:
> This is an app lifetime Cts, do we need to dispose it at all?

Yes, you are right that it is somewhat unnecessary. However, I had a discussion about this earlier when I worked on `TorHttpPool` and Lucas said we dispose everything correctly so I do so without exceptions.

Reply:
Ok, I agree, forget about this comment. We should dispose it.

Reply:
(Well, it's actually a very valid comment here but better to keep the convention.)